plugin/pkg/admission/podtolerationrestriction/admission.go

```diff
@@ -162,6 +166,23 @@ func (p *podTolerationsPlugin) Admit(a admission.Attributes) error {
 		}
 	}
 
+	if utilfeature.DefaultFeatureGate.Enabled(features.TaintNodesByCondition) {
```

Comment:
Similarly as in other PR - I think it's fine to add those tolerations even if feature is not enabled.